pkg/anypoint/client.go

```diff
@@ -148,19 +146,16 @@ func (c *AnypointClient) healthcheck(name string) (status *hc.Status) {
 
 // GetAccessToken retrieves a token
 func (c *AnypointClient) GetAccessToken() (string, *User, time.Duration, error) {
-	url := c.baseURL + "/accounts/login"
+	if c.clientID == "" || c.clientSecret == "" {
```

Comment:
Guessing that we are removing this with mulesoft removing the support for username/password based access token ?

Reply:
They didn't explicitly remove username/password access, but they require MFA with it, which makes it impossible for the agent to fulfill the auth process